tests/test_generate.py

```diff
@@ -60,7 +60,7 @@ def multinomial(*args, **kwargs):
     with mock.patch("torch.multinomial", multinomial):
         out = generate.generate(model, input_idx, max_new_tokens, max_seq_length=max_seq_length, top_k=4)
 
-    assert out.size(0) == T + max_new_tokens
+    assert out.size(0) >= T + max_new_tokens
```

Comment:
Why did this change? The generate function did not

Reply:
I fix it, sorry I made a mistake